com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -178,6 +178,8 @@ public partial class UniversalRenderPipelineAsset : RenderPipelineAsset, ISerial
         [SerializeField] float m_ShadowDepthBias = 1.0f;
         [SerializeField] float m_ShadowNormalBias = 1.0f;
         [SerializeField] bool m_SoftShadowsSupported = false;
+        [SerializeField] bool m_TightEnclosingSphere = false;
+        [SerializeField] int m_NumIterationsEnclosingSphere = 64;
```

Comment:
How is this determined. Are there some sort of error bounds or empirical testing?

Reply:
Just some manual empirical testing, and this gets the accuracy down to the 0.001.